runnumber-rest/rest.py

```diff
@@ -10,12 +10,13 @@
 import os
 
 from api import app, db
-
-# setenv DEBUG=True to enable debug mode
-DEBUG = os.getenv("DEBUG", "False").lower() == "true"
```

Comment:
Why revert this?

Reply:
This was a mistake when merging Pierre's PR

Reply:
Will change it back